src/software/world/field.h

```diff
@@ -182,13 +182,41 @@ class Field
      */
     Rectangle friendlyHalf() const;
 
+    /**
+     * Gets the friendly positive quadrant of the field
+     *
+     * @return the friendly positive quadrant of the field
+     */
+    Rectangle friendlyPositiveQuadrant() const;
```

Comment:
In general, could we say `PositiveY` instead of `Positive` (assuming I understand this correctly)? I just want to make sure we're being super unambiguous.

Reply:
Done.